echopype/convert/set_groups_ek80.py

```diff
@@ -150,14 +150,12 @@ def set_platform(self) -> xr.Dataset:
                         "valid_range": (-90.0, 90.0),
                     },
                 ),
-                "heave": (
+                "vertical_offset": (
                     ["mru_time"],
                     np.array(self.parser_obj.mru.get("heave", [np.nan])),
                     {
-                        "long_name": "Platform heave",
-                        "standard_name": "platform_heave_angle",
-                        "units": "arc_degree",
-                        "valid_range": (-90.0, 90.0),
+                        "long_name": "Platform vertical offset from nominal",
+                        "units": "m",
```

Comment:
For this part why wouldn't the same `self._varattrs` approach like in EK60 work?

Reply:
Agreed